--- a/config.def.h	2018-05-24 00:05:43.850599123 +0700
+++ b/config.def.h	2018-05-24 00:32:23.012298691 +0700
@@ -32,7 +32,7 @@
 /* layout(s) */
 static const float mfact     = 0.55; /* factor of master area size [0.05..0.95] */
 static const int nmaster     = 1;    /* number of clients in master area */
-static const int resizehints = 1;    /* 1 means respect size hints in tiled resizals */
+static const int resizehints = 0;    /* 1 means respect size hints in tiled resizals */
 
 static const Layout layouts[] = {
 	/* symbol     arrange function */
