--- a/config.def.h	2018-05-24 00:05:43.850599123 +0700
+++ b/config.def.h	2018-05-24 00:25:00.141412385 +0700
@@ -2,15 +2,15 @@
 
 /* appearance */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"Iosevka Term:size=12:antialias=true:autohint=false"
 };
-static const char dmenufont[]       = "monospace:size=10";
-static const char normbordercolor[] = "#444444";
-static const char normbgcolor[]     = "#222222";
-static const char normfgcolor[]     = "#bbbbbb";
-static const char selbordercolor[]  = "#005577";
-static const char selbgcolor[]      = "#005577";
-static const char selfgcolor[]      = "#eeeeee";
+static const char dmenufont[]       = "Iosevka Term:size=12:antialias=true:autohint=false";
+static const char normbordercolor[] = "#282828";
+static const char normbgcolor[]     = "#282828";
+static const char normfgcolor[]     = "#f9f5d7";
+static const char selbordercolor[]  = "#076678";
+static const char selbgcolor[]      = "#076678";
+static const char selfgcolor[]      = "#f9f5d7";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
